README.md

```diff
@@ -1,87 +1,142 @@
 ```
 
-1.6	Run podman images command to verify the repository and tag is correctly updated to the docker image
+Add the authentication details for the registry.connect.redhat.com registry to the ContainerImageRegistryCredentials parameter:
+
 ```
-sudo podman images
-REPOSITORY                                                                                                            TAG                                IMAGE ID               CREATED                    SIZE
-cld13b4.ctlplane.set.rdlabs.hpecorp.net:8787/rhosp-rhel8/openstack-cinder-volume-hpe                      latest                             d599f4f06967        2 minutes ago       1.18 GB
+parameter_defaults:
+  ContainerImageRegistryCredentials:
+    registry.redhat.io:
+      [service account username]: [service account password]
+    registry.connect.redhat.com:
+      [service account username]: [service account password]
 ```
 
-1.7	Push the container to a local registry
+Save the containers-prepare-parameter-hpe.yaml file.
+
+Include the containers-prepare-parameter-hpe.yaml file with any deployment commands, such as as openstack overcloud deploy:
+
 ```
-sudo openstack tripleo container image push --local cld13b4.ctlplane.set.rdlabs.hpecorp.net:8787/rhosp-rhel8/openstack-cinder-volume-hpe:latest
+openstack overcloud deploy --templates
+    ...
+    -e containers-prepare-parameter-hpe.yaml
+    ...
 ```
 
-### 2.	Prepare custom environment yaml
+When director deploys the overcloud, the overcloud uses the HPE container image instead of the standard container image.
+
+IMPORTANT:
+
+The containers-prepare-parameter-hpe.yaml file replaces the standard containers-prepare-parameter.yaml file in your overcloud deployment. Do not include the standard containers-prepare-parameter.yaml file in your overcloud deployment. Retain the standard containers-prepare-parameter.yaml file for your undercloud installation and updates.
+
+
+
+#### 1.2 Environment File for cinder backend
+
+The environment file is a OSP director environment file. The environment file contains the settings for each backend you want to define.
+
+Create the environment file “cinder-hpe-[iscsi|fc].yaml” under /home/stack/templates/ with below parameters and other backend details.
 
-Create new env file “custom_container_[iscsi|fc].yaml” under /home/stack/custom_container/ with only the custom container parameter and other backend details. Sample files are available in [custom_container](https://github.com/hpe-storage/hpe-3par-cinder-rhosp16.1/blob/master/custom_container) folder for reference
 ```
 parameter_defaults:
-    DockerCinderVolumeImage: cld13b4.ctlplane.set.rdlabs.hpecorp.net:8787/rhosp-rhel8/openstack-cinder-volume-hpe:latest
+  CinderEnableIscsiBackend: false
+  ControllerExtraConfig:
 ```
 
-### 3.	Deploy the overcloud
+Sample files for both iSCSI and FC backends are available in [templates](https://github.com/traghavendra/hpe-3par-cinder-rhosp16.1/tree/main/templates) folder for reference.
```

Comment:
Need to put master link here instead of forked repo user.

Reply:
done